File: Selecao I/multiplo.cpp
/*@multiplo: Múltiplo de sete*/

#include <iostream>
#include <locale>
using namespace std;

int main(){
    setlocale(LC_ALL, "Portuguese");
    
    int n; 
    cin >> n;

    if(n % 2 == 0){
        cout << "SIM" << endl;
    }else{
        cout << "NAO" << endl;
    }
    return 0;
}

File: src/Selecao_II/Multiplo.java
/**
 * Questão: @multiplo: Múltiplo de sete
 *
 * @author UxieGu1
 */
package Selecao_II;
import java.util.Scanner;

public class Multiplo {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        int n = scan.nextInt();

        if(n % 7 == 0) {
            System.out.println("SIM");
        } else {
            System.out.println("NAO");
        }

        scan.close();
    }
}
